x-pack/plugins/security_solution/public/management/pages/policy/store/policy_details/reducer/initial_policy_details_state.ts

```diff
@@ -38,6 +38,7 @@ export const initialPolicyDetailsState: () => Immutable<PolicyDetailsState> = ()
     trustedAppsToUpdate: createUninitialisedResourceState(),
     assignableListEntriesExist: createUninitialisedResourceState(),
     doesAnyTrustedAppExists: createUninitialisedResourceState(),
+    hasTrustedApps: createUninitialisedResourceState(),
```

Comment:
Likewise here, this probably should be named `hasTrustedAppsForPolicy`.

Reply:
yeah, this is the initial value for the type defined above